CIPs/CIP-23/CIP-23.md

```diff
@@ -0,0 +1,164 @@
+
+```json
+{
+  "name": "Social Accounts",
+  "description": "Social accounts linked to your DID",
+  "schema": "<reference-schema-DocID>",
+  "config": {
+    "tags": ["SocialAccounts"]
+  }
+}
+```
+
+### Reference Schema
+
+The reference schema defines a document which maintains an array of JSON objects that represent a DID's linked accounts. Below find the fields that are contained in each object.
+
+| Property       | Description                                                  | Value            | Max Size  | Required | Example                                      |
+| -------------- | ------------------------------------------------------------ | ---------------- | --------- | -------- | -------------------------------------------- |
+| `protocol`     | Network protocol used to resolve the host                    | string           | 50 char   | required | https                                        |
+| `host`         | Host used to resolve the ID                                  | string           | 150 char  | required | https://github.com                           |
+| `id`           | Path of the unique ID within the host. Can be the same as the host to represent the root. | string           | 450 char  | required | https://github.com/marysmith                 |
+| `claim`        | Unique location where the DID string is stored. Should be a place where only the given ID could have posted the DID. | string           | 450 char  | optional | https://gist.github.com/marysmith/5c48debdb7 |
+| `attestations` | Attestations issued by third-party services which have verified the claim against the ID. | array of strings | 1000 char | optional | ceramic://bafy...123                         |
+
+**Deployment:** `<reference-schema-DocID>`
+
+```jsx
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "array",
+  "title": "SocialAccounts",
+  "items": {
+    "$ref": "#/definitions/Account"
+  },
+  "definitions": {
+    "Account": {
+      "type": "object",
+      "properties": {
+        "protocol": {
+          "type": "string",
+		      "maxLength": 50
+        },
+        "host": {
+          "type": "string",
+		      "maxLength": 150
+        },
+        "id": {
+          "type": "string",
+		      "maxLength": 450
+        },
+        "claim": {
+          "type": "string",
+		      "maxLength": 450
+        },
+        "attestations": {
+          "type": "array"
+          "items": {
+            "type": "string",
+		      	"maxLength": 1000
+          }
+        }
+      },
+      "required": [
+        "protocol",
+        "host",
+        "id"
+      ]
+    }
+  }
+}
+```
+
+### Tags
+
+In the config of the *definition* defined above we can see a `tags` property. We simply apply the tags in the config to the `tags` property in the *metadata* of the *reference* Tile.
+
+### Example
+
+An example Crypto Accounts *reference* document that includes two Ethereum accounts and one Bitcoin account.
```

Comment:
This should not be crypto accounts. This should describe what's included in this definition.

Reply:
good catch, ty